src/components/Home/home.scss

```diff
@@ -0,0 +1,37 @@
+.link {
```

Comment:
be careful with naming things.  If we ever add anything else in an app in any file and call it link it'll override this one or this one will override that one.  A common naming convention is something like:
homeLink so we know the component it's suppose to be attached to (each file the way we have it set up currently is associated with exactly one file)